### DISCUSSION TITLE
Is this a bug? (IRQ0 not handled)

### DISCUSSION THREAD

**Discussion by guoshimin** [oxidecomputer/hubris#367] [Q&A] [answered]
In this line https://github.com/oxidecomputer/hubris/blob/a13fce543958a7bd5fa06e217cd2f28e139deb72/sys/kern/src/arch/arm_m.rs#L956, should it be `x >= 16`, so that exception_num 16, aka IRQ0, is accepted?

**Comment by mx-shift** [comment, marked as answer]
Indeed it is!  Thanks for finding it.
